server/src/main/java/org/apache/druid/server/coordination/SegmentLoadDropHandler.java

```diff
@@ -338,8 +339,8 @@ private List<DataSegment> getBootstrapSegments()
     finally {
       stopwatch.stop();
       final long fetchRunMillis = stopwatch.millisElapsed();
-      emitter.emit(new ServiceMetricEvent.Builder().setMetric("bootstrapSegments/fetch/time", fetchRunMillis));
-      emitter.emit(new ServiceMetricEvent.Builder().setMetric("bootstrapSegments/fetch/count", bootstrapSegments.size()));
+      emitter.emit(new ServiceMetricEvent.Builder().setMetric("segment/bootstrap/time", fetchRunMillis));
+      emitter.emit(new ServiceMetricEvent.Builder().setMetric("segment/bootstrap/count", bootstrapSegments.size()));
```

Comment:
[Non blocker] It would be nice to add the `dataSource` dimension to this metric.

Reply:
Good idea, I will save this for a future patch